R/pkg/R/SQLContext.R

```diff
@@ -75,8 +75,8 @@ infer_type <- function(x) {
 #'\dontrun{
 #' sc <- sparkR.init()
 #' sqlContext <- sparkRSQL.init(sc)
-#' rdd <- lapply(parallelize(sc, 1:10), function(x) list(a=x, b=as.character(x)))
-#' df <- createDataFrame(sqlContext, rdd)
+#' df1 <- as.DataFrame(sqlContext, iris)
```

Comment:
what does iris mean?

We'd better keep original examples for createDataFrame, and add new examples for as.DataFrame